### DISCUSSION TITLE
Genmon is unable to start my generator

### DISCUSSION THREAD

**Discussion by raymondjclark** [jgyates/genmon#1082] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

V1.19.00

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

I used to be able to start the generator. It is at a remote site without utility power. The solar panel to keep the battery charged got damaged and the battery died. After replacing the battery Genmon no longer is able to start the generator. Are there any debugging tools that might explain what got messed up?

**Comment by jgyates** [comment]
I got your logs. Nothing looks out of the ordinary. 

Bason on the registers including in your logs, modbus registers 236 shows that you are in two wire mode. This is typically how off grid setups start the generator. It is my understanding that remote commands do not work in two wire mode. The question may be, why did it work before and changing the battery restarted the controller and now the fluke with the controller responding to remote commands in two wire mode is gone.

one thing you could try is to disable two wire mode, then see if the remote commands come back, then re-enabled it (without rebooting the controller) and see if the remote functions remain. 

Another think to try is to disable two wire, then remove power to the controller or reboot it with two wire disabled, then validate remote start works, then re-enable two wire.

The controller may disable remote commands when it boots if two wire is enabled, but not disable it if two wire is enabled after the boot.

This page details how to power cycle the controller:

https://github.com/jgyates/genmon/wiki/Appendix-I-Power-Cycle-the-Evolution-or-Nexus-Controller

**Comment by raymondjclark** [comment]
Thanks, we got over 2 feet of snow so it will be a few days before I can get to the generator.RayRaymond ClarkSent from my iPhoneExpect tYopSOn Mar 14, 2024, at 10:57 AM, jgyates ***@***.***> wrote:﻿
I got your logs. Nothing looks out of the ordinary.
Bason on the registers including in your logs, modbus registers 236 shows that you are in two wire mode. This is typically how off grid setups start the generator. It is my understanding that remote commands do not work in two wire mode. The question may be, why did it work before and changing the battery restarted the controller and now the fluke with the controller responding to remote commands in two wire mode is gone.
one thing you could try is to disable two wire mode, then see if the remote commands come back, then re-enabled it (without rebooting the controller) and see if the remote functions remain.
Another think to try is to disable two wire, then remove power to the controller or reboot it with two wire disabled, then validate remote start works, then re-enable two wire.
The controller may disable remote commands when it boots if two wire is enabled, but not disable it if two wire is enabled after the boot.
This page details how to power cycle the controller:
https://github.com/jgyates/genmon/wiki/Appendix-I-Power-Cycle-the-Evolution-or-Nexus-Controller

**Comment by jgyates** [comment]
ok. Let me know if that resolves your issue. That would be good info to have.

**Comment by raymondjclark** [comment]
Absolutely Raymond ClarkSent from my iPhoneExpect tYopSOn Mar 14, 2024, at 11:11 AM, jgyates ***@***.***> wrote:﻿
ok. Let me know if that resolves your issue. That would be good info to have.

**Comment by raymondjclark** [comment]
I disabled 2 wire start and rebooted the controller and still could not start it. As a side note I reenabled 2 wire start and it doesn't work either. Maybe the controller is fried? It starts in both manual and auto mode.

**Comment by jgyates** [comment]
I guess that is possible. I don't know enough about the controller internals to know what would cause remote start functionality not to work. You may of done this but if may be worth another test to disable 2-wire mode, and disconnect the two wires used to start the controller. I don't know if that input has a bias resistor that would make the controller read a different value if it were not connected. It *should* ignore that input if two wire is disabled, but it is not behaving as expected so I am just guessing here.
